exercises/reverse-string/README.md

```diff
@@ -6,6 +6,19 @@ For example:
 input: "cool"
 output: "looc"
 
+## Bonus
+
+Preserve grapheme clusters, i.e.
+
+```julia
+myreverse("hi 👋🏾") == "👋🏾 ih"
+myreverse("as⃝df̅") == "f̅ds⃝a"
+```
+
+You will probably find the `Unicode` stdlib useful for this bonus task.
+
+To enable the graphemes test, add `const TEST_GRAPHEMES = true` to the global scope of your file.
```

Comment:
I wish Julia had a nicer way of handling this.

Reply:
I don't think there's much that can be done. Test or build suites in other languages would read a DEFINE or config files or environment variables. We want the student to only have to modify the one file and we want mentors to run the same tests the students are running, so we probably don't want a config file or env variables.

We could define a function in runtests and give the students an API like

```julia
enable_bonus_test(:graphemes)
```

or something?

That would require some changes to our repo-level runtests because that includes the files in a different order.

Reply:
I think how you did it works fine, it would just be nice if Julia had a nice way to run specific testsets. (there's an issue open for that I think)